libfwupdplugin/fu-udev-device.c

```diff
@@ -1541,15 +1541,19 @@ fu_udev_device_set_io_channel(FuUdevDevice *self, FuIOChannel *io_channel)
  * Since: 1.3.6
  **/
 void
-fu_udev_device_set_flags(FuUdevDevice *self, FuUdevDeviceFlags flags)
+fu_udev_device_add_flag(FuUdevDevice *self, FuUdevDeviceFlags flag)
```

Comment:
presumably there should be a matching remove flag too with this.

Reply:
Fixed, thanks.